### DISCUSSION TITLE
optimized Layer time

### DISCUSSION THREAD

**Discussion by nickeeey** [j7126/OctoPrint-Dashboard#277] [Ideas]
Hi, i really like to watch my layer time. Maybe can you improve it? Mostly i print with brim and so my first layer time messes up the chart and i can't see very much (see screenshot).
Maybe you do (if possible) an interactive chart with selecting an Range
![image](https://user-images.githubusercontent.com/22114715/118397809-30606b00-b656-11eb-825a-23aad11a0679.png)

**Comment by j7126** [comment]
Hi @nickeeey, that is actually already an option.

Go to settings -> dashboard -> layer duration graph -> options -> Layer graph type, and set it to last 40 layers.

I do agree that there is some room for improvement here by allowing changing of last 40 layers to a custom number of layers, and I will look into this.

**Reply by nickeeey** [reply]
Thanks, i oversaw the settings there in the menu. Now its much better ;)

**Comment by StefanCohen** [comment]
There are some additional improvements in #105 that are still valid. For example a non-linear Y-scale, a Y cut-off. I've also been thinking of a setting to exclude the first n layers. That is probably the simplest solution technically. Just an offset on the layer duration array.

Edit. Holy s**t. Is that 50 minutes for your first layer? You must be very patient :)
